hacking/tagger/tag.py

```diff
@@ -41,7 +41,12 @@
 
 DEFAULT_ANSIBLE_CORE_CHECKOUT = ROOT.parent.joinpath("ansible")
 DEFAULT_REMOTE = "origin"
-DEFAULT_ACTIVE_BRANCHES: tuple[str, ...] = ("stable-2.14", "stable-2.15", "stable-2.16")
+DEFAULT_ACTIVE_BRANCHES: tuple[str, ...] = (
+    "stable-2.14",
+    "stable-2.15",
+    "stable-2.16",
+    "stable-2.17",
+)
```

Comment:
This one is a bit beyond me, but based on the title (DEFAULT ACTIVE BRANCHES) - it seems safe to merge immediately. But I'd get @gotmax23 review here to be sure.

Then after 2.14 EOLs I'd guess we remove that one?

Reply:
See the comment here: https://github.com/ansible/ansible-documentation/pull/1251#discussion_r1552276502